src/backend/src/prisma/schema.prisma

```diff
@@ -640,6 +655,45 @@ model Meeting {
   teamId       String
 }
 
+model Wbs_Proposed_Changes {
+  wbsProposedChangesId       String                         @id @default(uuid())
+  name                       String
+  status                     WBS_Element_Status
+  projectLeadId              Int
+  projectManagerId           Int
+  links                      Link[]                         @relation(name: "proposedChangeLinks")
+  changRequestId             Int                            @unique
+  changeRequest              Change_Request                 @relation(fields: [changRequestId], references: [crId])
+  proposedProjectChanges     Project_Proposed_Changes?      @relation(name: "proposedProjectChanges")
+  workPackageProposedChanges Work_Package_Proposed_Changes? @relation(name: "wpProposedChanges")
+}
+
+model Project_Proposed_Changes {
+  projectProposedChangesId String               @id @default(uuid())
+  budget                   Int
+  summary                  String
+  rules                    String[]
+  goals                    Description_Bullet[] @relation(name: "proposedProjectGoals")
+  features                 Description_Bullet[] @relation(name: "proposedProjectFeatures")
+  otherConstraints         Description_Bullet[] @relation(name: "proposedProjectOtherConstraints")
+  teams                    Team[]               @relation(name: "proposedProjectTeams")
+  proposedWbsChanges       Wbs_Proposed_Changes @relation(name: "proposedProjectChanges", fields: [wbsProposedChangesId], references: [wbsProposedChangesId])
+  wbsProposedChangesId     String               @unique
+}
+
+model Work_Package_Proposed_Changes {
+  workPackageProposedChangesId String               @id @default(uuid())
+  orderInProject               Int
```

Comment:
remove this field, we shouldn't ever be editing it after the work package is created (ticket's wrong)